packages/connect-webextension/src/index.ts

```diff
@@ -196,6 +196,40 @@ const TrezorConnect = factory({
     dispose,
 });
 
+const _channel = new ServiceWorkerWindowChannel<any>({
```

Comment:
what happens if this file is imported multiple types? woudn't it create a new instance and do some unexpected behavior?

Reply:
it is pretty typical usecase to import from index multiple times in background serviceworker

Reply:
Ok, maybe having it in a different file as I had before was better, so the developer can add it or not.

Reply:
I am basically just interested whether it can handle some more complex scenarios before we push it out of beta

Reply:
I understand, that should not be an issue, that code only runs when file is loaded, as it does the creation of the TrezorConnect `factory` that we have in the same file.
But in order to make it cleaner I encapsulated it in a function like https://github.com/trezor/trezor-suite/pull/10751/commits/59386c35f347e46034f120d444a20d62b0490fe4


We could make the developer to run an function that initiates that but I don't think it will be less friendly and we want that to run as soon as possible so the service worker is ready to receive hankshake request from front-end.